src/config.c

```diff
@@ -611,6 +611,12 @@ void loadServerConfigFromString(char *config) {
         goto loaderr;
     }
 
+    /* in case cluster mode is enabled dbnum must be 1 */
+    if (server.cluster_enabled && server.dbnum > 1){
+        serverLog(LL_WARNING, "changing databases number from %d to 1 since we are in cluster mode", server.dbnum);
```

Comment:
```suggestion
    if (server.cluster_enabled && server.dbnum > 1) {
        serverLog(LL_WARNING, "WARNING: Changing databases number from %d to 1 since we are in cluster mode", server.dbnum);
```

Reply:
It's already a warning, why add more text saying it's a warning?

Reply:
it's a warning in the log level filtering, and it has the `#` prefix for the line, but people who look at the log with a plain text viewer don't really notice these differences (can be improved if we merge #10650).
there are many other places in which our important log prints starts with a bold "WARNING" string.